core/actions/table.ts

```diff
@@ -553,7 +562,19 @@ export class Table extends ActionBuilder<dataform.Table> {
   public dependencies(value: Resolvable | Resolvable[]) {
     const newDependencies = Array.isArray(value) ? value : [value];
     newDependencies.forEach(resolvable => {
-      this.proto.dependencyTargets.push(resolvableAsTarget(resolvable));
+      let dependencyTarget = resolvableAsTarget(resolvable);
+      dependencyTarget.includeDependentAssertions = dependencyTarget.includeDependentAssertions===undefined ? this.dependOnDependencyAssertions : dependencyTarget.includeDependentAssertions;
```

Comment:
This is logically identical to

```suggestion
      dependencyTarget.includeDependentAssertions = dependencyTarget.includeDependentAssertions || this.dependOnDependencyAssertions;
```

Reply:
this will not work. we want to prioritize the `dependencyTarget.includeDependentAssertions` over `this.dependOnDependencyAssertions` regardless of value itself(true or false). only in case this is not given we use the default setting `this.dependOnDependencyAssertions`.